### PR TITLE
Add all markdown files to the linter

### PR DESCRIPTION
**Pull request by rdavid**

### CODE REVIEW

File: docs/configuration.md [hunk 1]
```diff
@@ -51,24 +57,26 @@ targetCrdFilters:
...
+- If it is not present, then only known IPs associated with the pod are used (the pod `.status.ips` field).
 
 #### Network Interfaces
 
-* The `k8s.v1.cni.cncf.io/networks-status` annotation is checked and the `interface` from the first entry found with `"default"=true` is used. This annotation is automatically managed in OpenShift but may not be present in K8s.
+- The `k8s.v1.cni.cncf.io/networks-status` annotation is checked and the `interface` from the first entry found with `“default”=true` is used. This annotation is automatically managed in OpenShift but may not be present in K8s.
 
-The label `test-network-function.com/skip_connectivity_tests` excludes pods from all connectivity tests. The label value is not important, only its presence.
-The label `test-network-function.com/skip_multus_connectivity_tests` excludes pods from [Multus](https://github.com/k8snetworkplumbingwg/multus-cni) connectivity tests. Tests on default interface are still done. The label value is not important, but its presence.
+The label `test-network-function.com/skip_connectivity_tests` excludes pods from all connectivity tests. The label value is trifling, only its presence.
```
Comment (sebrandon1) [inline docs/configuration.md:74]: I'd rather this stay "not important".  "Trifling" makes no sense.

**Review by sebrandon1 — CHANGES_REQUESTED** [review]
One comment about word usage.  Also, does `make build-catalog-md` break any of this when ran?

File: docs/configuration.md (hunk 1 above)
Comment (rdavid) [reply docs/configuration.md:74]: `not important` triggers the linter. Other option?

Comment (rdavid) [reply docs/configuration.md:74]: >does make build-catalog-md break any of this when ran
I guess it doesn't, I'm going to check.

Comment (bnshr) [reply docs/configuration.md:74]: trivial is also an alternative.

Comment (rdavid) [reply docs/configuration.md:74]: @sebrandon1 Is `The label value is trivial` good to go?

Comment (sebrandon1) [reply docs/configuration.md:74]: Yes, `trivial` makes more sense to me.

Comment (rdavid) [reply docs/configuration.md:74]: @sebrandon1 Fixed `make build-catalog-md`.

File: docs/configuration.md [hunk 2]
```diff
@@ -51,24 +57,26 @@ targetCrdFilters:
...
+The label `test-network-function.com/skip_multus_connectivity_tests` excludes pods from [Multus](https://github.com/k8snetworkplumbingwg/multus-cni) connectivity tests. Tests on default interface are still done. The label value is trifling, but its presence.
```
Comment (sebrandon1) [inline docs/configuration.md:75]: This is still `trifling`?

**Review by sebrandon1 — COMMENTED** [review]
Just a comment

File: docs/test-container.md [hunk 3]
```diff
@@ -117,12 +119,13 @@ podman build -t cnf-certification-test:v4.1.7 \
   --build-arg TNF_SRC_URL=https://github.com/test-network-function/cnf-certification-test .
 ```
 
-### Run the tests
+### Run the tests 2
```
[the remark this answers is not in the document]
Comment (rdavid) [reply docs/test-container.md:122]: The linter doesn't allow paragraphs with the same name.

**Review by sebrandon1 — APPROVED** [review]

**Pull request merged**